userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/Constants.java

```diff
@@ -72,7 +72,7 @@ public String getText() {
     }
 
     enum RedirectUri {
-        Regular("msauth://example.adal.aad.microsoft.com.userappwithbroker/IcB5PxIyvbLkbFVtBI%2FitkW%2Fejk%3D"),
+        Regular("msauth://com.microsoft.aad.adal.userappwithbroker/IcB5PxIyvbLkbFVtBI%2FitkW%2Fejk%3D"),
```

Comment:
@addev-ashish This is a correction to the supplied redirect uri

Reply:
In addition, the signature in the redirect uri might change in each build.

Reply:
no it should not